protos/flyteidl/admin/launch_plan.proto

```diff
@@ -90,7 +91,13 @@ message LaunchPlanSpec {
     // Indicates the permission associated with workflow executions triggered with this launch plan.
     Auth auth                             = 8 [deprecated=true];
 
-    AuthRole auth_role                    = 9;
+    AuthRole auth_role                    = 9 [deprecated=true];
+
+    // Indicates security context for permissions triggered with this launch plan
+    SecurityContext security_context      = 10;
+
+    // Map of tasktype to overrideable for each task type.
```

Comment:
is the key value here task type name?

Reply:
Yeah. I just followed the pattern here: https://github.com/flyteorg/flyteidl/blob/master/protos/flyteidl/admin/matchable_resource.proto#L69